### PR TITLE
add address checking

### PR DESCRIPTION
**Pull request by frosty00**
Somehow my IDE is dropping semicolons everywhere - I don't know how to stop that, sorry. Anyhow I thought it would be nice to have a seperate error related being sent a bad address in `fetchDepositAddress` or `createDepositAddress`, as we don't want our users to lose all their crypto by sending it to a bad address by accident.

### needs to be translated into php

### CODE REVIEW

**Comment by kornrunner** [comment]
I can help out with PHP if needed

**Comment by frosty00** [comment]
@kornrunner awesome.

https://github.com/ccxt/ccxt/pull/2138/files#diff-8202271aa00266e981baed72b24c75eeR836

This method and a error called "InvalidAddress" need to be added. Cheers

**Comment by kroitor** [comment]
Thanks, @frosty00, @kornrunner ! )

BTW, one of the ways to convert to php is to actually implement it in js/py and then use the transpiler to do the job ) From there it remains to copy the generated method to the base class and that's it ) Anyway, thanks for your help! Will merge shortly.

**Comment by kroitor** [comment]
oh... @frosty00 i've restored them already... %)

**Comment by frosty00** [comment]
I'm confused as to why they aren't in the base file? Is it still sentatically correct javascript? Are they just used for transpilling? What does webpack and babel do? Sorry for the bombardment of questions..

**Comment by kroitor** [comment]
> I'm confused as to why they aren't in the base file?

Well, because, in general, JS does not need them )

>  Is it still sentatically correct javascript?

Yep, it is )

> Are they just used for transpilling?

For transpiling to PHP to be precise, PHP requires them, and they are also used as hints for general transpiling of multi-line expressions as well. In general, it's muuuuch easier to remove the unnecessary semicolons, then to add them where they are necessary but missing. Thus we remove them upon transpiling to Python, not add them upon transpiling to PHP. And JS does not care at all (mostly), it is still syntactically correct with or without the semicolons.

> What does webpack and babel do?

In short, they transform newer code syntax (supported in node) to the older flavors of it, supported in, say, your browser. And they abstract it somewhat, for example, in node you have a file system, like you do in python, but in a browser you don't have any files, except URLs, etc. Webpack and babel are mostly used to address cross-compatibility issues between different environments. Anyway, nowadays, most browsers and node can handle ES6. 

> Sorry for the bombardment of questions...

No worries at all, feel free to ask any time )

**Comment by frosty00** [comment]
The first commit of the PR is already ready to be merged if you don't mind me adding a couple hundred semi's (: 

33b43ce8d815253285a445557fbdb64e7f2dee7c
